lib/media/drm_engine.js

```diff
@@ -698,7 +698,7 @@ shaka.media.DrmEngine = class {
     // Create a config entry for each key system.
     for (const info of allDrmInfo) {
       const config = {
-        // Ignore initDataTypes.
+        initDataTypes: ['cenc'],
```

Comment:
Hi @avelad 👋   For it to be complete and prevent regressions, I think you should also take the values from initData, in case of other initDataTypes (I can think of `sinf` for example, when working with hls fmp4 fairplay 2 & 3), you could take it from here https://github.com/google/shaka-player/pull/2750/files#diff-003386d961d9c0958312b9b16780a8eb91f8c57ecb8e04810a950af8819db4a2R741

Reply:
I will add it

Reply:
Done!